.github/workflows/CI-DOCKER-DPDK.yml

```diff
@@ -0,0 +1,52 @@
+name: "p4c-dpdk-ptf-p4testgen-tests"
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+      
+concurrency:
+  # If workflow for PR or push is already running stop it, and start new one.
+  group: p4c_dpdk_ptf_ci-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build_p4dpdk_ubuntu:
+    runs-on: ubuntu-22.04
+    steps:
+    # TODO: Cache for container
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@v1
+        with:
+          key: ptf-${{ runner.os }}-test
+          max-size: 1000M
+
+      - name: Checkout P4C
+        uses: actions/checkout@v3
+        with:
+             path: p4c
+             submodules: recursive
+
+      - name: Checkout DPDK-target
+        uses: actions/checkout@v3
+        with:
+             repository: p4lang/p4-dpdk-target
+             path: p4sde
+             submodules: recursive
+
+      - name: Build IPDK
+        run: |
+             cp p4c/tools/dpdk_ptf.Dockerfile .
```

Comment:
Why copy the file instead of just specifying the path? Is there a particular problem with the setup?

Reply:
It is for simplification in the Dockerfile where the folders, p4c and p4sde, can be copied from the current directory directly without having to go to parent directory for them.